src/main/java/org/isf/admission/gui/AdmissionBrowser.java

```diff
@@ -531,7 +531,7 @@ private JTabbedPane getJTabbedPaneAdmission() {
 					jTabbedPaneAdmission.setEnabledAt(pregnancyTabIndex, false);
 				}
 			}
-			jTabbedPaneAdmission.addTab("Note", getJPanelNote());
+			jTabbedPaneAdmission.addTab(MessageBundle.getMessage("angal.common.plain.title"), getJPanelNote());
```

Comment:
should it be "angal.common.note.txt"?

Reply:
No; I've been making make dialogs and tabs use "title"; I'm thinking of switching "border" type to "title" too to reduce the number of different choices.

Reply:
FWIW, "plain.title" is "Note".

Reply:
I was wondering why "Note" had a key like "plain.title"

Reply:
per comment:    I've been making make dialogs and tabs use "title

Reply:
and it is called plain because that is what Java uses for titles for confirmation dialogs

Reply:
> and it is called plain because that is what Java uses for titles for confirmation dialogs

Sorry I don't get
![image](https://user-images.githubusercontent.com/2938553/121498308-971a3f80-c9dc-11eb-972a-1f36dd44bd99.png)

Reply:
The first parm to `addTab()` is the title on the tab; here is the method definition:
![image](https://user-images.githubusercontent.com/1105445/121501355-29a2ea80-c9ad-11eb-985d-c4cacc982691.png)

So the first parm is the "title" or the label on the tab and that is why I'm using ".title".

Reply:
I'm not questioning the suffix .title, but the name "plain"! 😂
since we are talking about the "Note" tab and not the "plain" tab, it should be "angal.common.note.title", like for others:

Admission/Discharge -> angal.admission.admissionanddischarge.title
Operation -> angal.admission.operation.title
Delivery -> angal.admission.delivery.title
**Note -> angal.common.note.title**

I don't see the good to call it plain, differently from other similar cases

```
angal.common.error.title                                                         = Error
angal.common.warning.title                                                    = Warning
angal.common.info.title                                                           = Information
angal.common.question.title                                                   = Question
angal.common.plain.title                                                         = Note
```

Reply:
I will change it but I was just following the statics in JOpitonPane.java
![image](https://user-images.githubusercontent.com/1105445/121512069-53f9a580-c9b7-11eb-9ffa-b68de45b4e90.png)

Reply:
changed pushed.

Reply:
Thanks. We are free to have both if we like:

angal.common.plain.title = Note
angal.common.note.title = Note

moreover, if the error, warning, info, question and plain labels are meant to be used only in that MessageDialog class, we can remove from "common" because not needed elsewhere.

Reply:
we could; but I do know that error and question are used currently in places other than MessageDialog

Reply:
OK; I've added "messagedialog.error/warning/info/question/note.title keys.   I also left common.error/info.title keys as they are used else where including the CORE repo.

This should be the final change, agreed?